presto-cli-rpm/pom.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
How long does it take to run `presto-cli-rpm` and `presto-cli-provisio`?

Reply:
<10 seconds for provisio and ~2m for presto-cli-rpm with tests (~10 seconds without tests).